src/sentry/analytics/event.py

```diff
@@ -116,5 +116,8 @@ def serialize(self):
     def from_instance(cls, instance, **kwargs):
         values = {}
         for attr in cls.attributes:
-            values[attr.name] = (kwargs.get(attr.name) or getattr(instance, attr.name, None))
+            try:
```

Comment:
this could also be implemented as `kwargs.get(key, getattr(instance, key, None))` but is also OK as is

Reply:
i like your approach better.